webapp/channels/src/utils/utils.tsx

```diff
@@ -502,19 +502,19 @@ export function applyTheme(theme: Theme) {
             'sidebar-text-30': changeOpacity(theme.sidebarText, 0.3),
             'sidebar-text-40': changeOpacity(theme.sidebarText, 0.4),
             'sidebar-text-50': changeOpacity(theme.sidebarText, 0.5),
-            'sidebar-text-60': changeOpacity(theme.sidebarText, 0.6),
-            'sidebar-text-72': changeOpacity(theme.sidebarText, 0.72),
+            'sidebar-text-60': changeOpacity(theme.sidebarText, 0.75),
```

Comment:
I also wouldn't expect these to change since we now have variables where the opacity doesn't match the name of them